.github/workflows/deploy.yml

```
@@ -0,0 +1,35 @@
+name: Deploy sdk-example
```

In theory we do not do a deploy here, it is more like triggering internal build. I would suggest to change it to 
```suggestion
name: Trigger build
```

Also the name of file can be changed :)